ocaml/idl/datamodel.ml

```diff
@@ -3158,8 +3158,8 @@ let host_reboot = call
     ~allowed_roles:_R_POOL_OP
     ()
 
-let host_prepare_for_shutdown = call
-    ~name:"prepare_for_shutdown"
+let host_prepare_for_poweroff = call
+    ~name:"prepare_for_poweroff"
```

Comment:
We should probably squash all the rename and refactor commits into one before merging and merge the commits messages having both our signoffs on it.

Reply:
Done